go.mod

```diff
@@ -4,15 +4,15 @@ go 1.16
 
 require (
 	github.com/go-logr/logr v0.2.1
+	github.com/mattn/go-isatty v0.0.12 // indirect
```

Comment:
Is this lib approved by CNCF ?

Reply:
This is the indirect library that is not used in HWCC.
But used by controller-gen from sigs.k8s.io/controller-tools which is Kubernetes library.
This is the flow of the library.

- github.com/metal3-io/hardware-classification-controller
-- sigs.k8s.io/controller-tools/cmd/controller-gen
--- sigs.k8s.io/controller-tools/pkg/genall/help/pretty
---- github.com/fatih/color

Reply:
okay. Looks good.